packages/labs/test/multiSelectTests.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ * Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+ * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+ * and https://github.com/palantir/blueprint/blob/master/PATENTS
+ */
+
+import { Classes, MenuItem } from "@blueprintjs/core";
+import { assert } from "chai";
+import * as classNames from "classnames";
+import { mount } from "enzyme";
+import * as React from "react";
+
+import { Film, TOP_100_FILMS } from "../examples/data";
+import { IMultiSelectProps, ISelectItemRendererProps, MultiSelect } from "../src/index";
+
+describe("<MultiSelect>", () => {
```

Comment:
The bug report is specifically about selecting an option when something is in `inputProps`. The bad behavior doesn't occur until the actual click occurs. I added an earlier test I removed that checks that `placeholder` is properly propagated but it would have passed before this fix as well.